miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -243,6 +267,36 @@ open class MiniAppMessageBridge {
         bridgeExecutor.postError(callbackObj.id, "${ErrorBridgeMessage.ERR_UNIQUE_ID} ${e.message}")
     }
 
+    private fun onGetContactId(callbackObj: CallbackObj) = try {
```

Comment:
See the updates on the iOS side - basically we wanted to rename this to `getMessagingUniqueId`.

Reply:
addressed 👍